includes/admin/models/class-wppr-review-model.php

```diff
@@ -910,6 +932,18 @@ public function get_small_thumbnail() {
 		return apply_filters( 'wppr_thumb', isset( $image_thumb[0] ) ? $image_thumb[0] : $this->image, $this->ID, $this );
 	}
 
+	/**
+	 * Return the review image's alt text.
+	 *
+	 * @since   3.4.3
+	 * @access  public
+	 * @return string
+	 */
+	public function get_wppr_image_alt() {
+		$alt = get_post_meta( $this->get_wppr_image_id(), '_wp_attachment_image_alt', true );
```

Comment:
@UVLabs add an empty check for image_id like on line 922

Reply:
I added a fall back for the ALT text, if for whatever reason the image ID is faulty then the $alt variable would empty string returned by get_post_meta() so the plugin would just fall back to the post name

Reply:
@UVLabs @contactashish13 is right, you should  the empty check, if the image id is faulty, get_post_meta returns false not empty string. you are echoing false in that alt context.

You can check here the behaviour when the post_id is empty -> https://core.trac.wordpress.org/browser/tags/4.9.6/src/wp-includes/meta.php#L461

Reply:
also, why you dont name the method, get_image_alt, instead of using wppr ? we don't have the risk of name colision as this is used inside the class.